Formula/proxysql.rb

```diff
@@ -0,0 +1,37 @@
+class Proxysql < Formula
+  desc "High Performance Advanced Proxy for MySQL"
+  homepage "http://www.proxysql.com/"
+  url "https://github.com/sysown/proxysql/archive/v1.4.10.tar.gz"
+  sha256 "29020cba75b778ef45b06c5469ee5930ce80afbd0290a112d8c2a6652b4c8a2e"
+
+  # Build dependencies listed here: https://github.com/sysown/proxysql/blob/master/INSTALL.md
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "mysql@5.6" => :test
```

Comment:
Is there no support for the latest `mysql`?

Reply:
Tests are fixed now on normal mysql.